src/App.tsx

```diff
@@ -15,36 +15,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  DEFAULT = 'default',
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+}
+
+const preparedGoods = [...goodsFromServer];
+
+const sortGoods = (order: SortType, isReversed: boolean): string[] => {
+  let sortedGoods: string[];
+
+  switch (order) {
```

Comment:
u can use switch inside of sorting function and sort only one time. for default case just return 0;